tide_site.install

```diff
@@ -185,3 +185,28 @@ function tide_site_update_8006() {
     $entity_form_display->save();
   }
 }
+
+/**
+ * Enable the field_media_site field in form_display of all media bundles.
+ */
+function tide_site_update_8007() {
+  // Default settings.
+  $default_option = [
+    'type' => 'options_buttons',
+    'region' => 'content',
+    'settings' => [],
+    'third_party_settings' => [],
+  ];
+  $bundle_info_service = \Drupal::service('entity_type.bundle.info');
+  $entity_form_display_storage = \Drupal::entityTypeManager()->getStorage('entity_form_display');
+  $media_bundles = $bundle_info_service->getBundleInfo('media');
+  if ($media_bundles) {
```

Comment:
Just a small comment about style, I prefer the condition to be explicit. Is `$media_bundles` a boolean or an array (an array if you look further) so something more explicit is preferable.

Reply:
Hi @anthony-malkoun 
```
if (!empty($media_bundles) && is_array($media_bundles)) {
```
will be more explicit, i think